docs/CHANGELOG.rst

```diff
@@ -0,0 +1,49 @@
+##########
+Changelog
+##########
+
+All notable changes to this project are documented in this file.
+This project adheres to `Semantic Versioning <http://semver.org/>`_.
+
+==========
+Unreleased
+==========
+
+==========
+0.2.0
+==========
```

Comment:
Align with length of string in the middle.

Reply:
updated